.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Uptane Standard CI Job'
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build and publish artefacts
+        env: 
+          ACCESS_TOKEN: ${{ secrets.GITHUB_ACCESS_TOKEN }}
+        uses: docker://advancedtelematic/uptane-ci
```

Comment:
Could we please move the CI image to the Uptane org?

Reply:
Sure! Except that it didn't exist until 10 seconds ago. It does now, though: https://hub.docker.com/u/uptane

Reply:
Ah, I see, thanks!

Just so maintenance is easier going forward, we should use [GitHub Package Registry](https://github.com/features/package-registry) instead, but it's in beta right now. I signed up to receive news about it, so will keep you informed.